exampleDataTrack/src/main/java/com/twilio/video/examples/datatrack/DataTrackActivity.java

```diff
@@ -46,7 +46,7 @@
 
     // You must provide a Twilio Access Token to connect to the Video service
     private static final String TWILIO_ACCESS_TOKEN = BuildConfig.TWILIO_ACCESS_TOKEN;
-    private static final String ACCESS_TOKEN_SERVER = BuildConfig.ACCESS_TOKEN_SERVER;
+    private static final String ACCESS_TOKEN_SERVER = BuildConfig.TWILIO_ACCESS_TOKEN_SERVER;
```

Comment:
Should we change the var to this name as well?

Reply:
I'm cool with that. Will update